src/Components/SignupBox/SignupBox.jsx

```diff
@@ -0,0 +1,154 @@
+import { useState, useEffect } from "react";
+import * as styles from "./SignupBox.module.css";
+
+export const SignupBox = ({ setAuth }) => {
+  //   useDocTitle("Signup | StormKeep");
+  //   const { signup } = useAuthContext();
+  const [userDetails, setUserDetails] = useState({
+    user: "",
+    email: "",
+    passwordOne: "",
+    passwordTwo: "",
+    terms: false,
+  });
+  const [error, setError] = useState({ state: false, text: "" });
+  const [type, setType] = useState(true);
+  const [trigger, setTrigger] = useState(true);
+
+  const validation = /^(?=.*\d)(?=.*[a-z]).{5,10}$/;
+  const emailValidation =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
suggestion: add to util, so that you can reuse

Reply:
it only being used in 2 files so i thought there is no need of a util file